x-pack/legacy/plugins/ml/public/components/kql_filter_bar/filter_bar/filter_bar.js

```diff
@@ -169,7 +169,7 @@ export class FilterBar extends Component {
               'xpack.ml.explorer.kueryBar.filterPlaceholder',
```

Comment:
Does this need to be a translation any more? Aren't you just passing the `placeholder` prop straight through now?

Reply:
It does not - removed in c6a1e15c98cb28c5aedb9f6530a0ded4e8cd73b9